tests/e2e/sync_test.go

```diff
@@ -0,0 +1,114 @@
+//go:build e2e_tests
+// +build e2e_tests
+
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"reflect"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+var DECK = filepath.Join("../../deck")
+
+func getKongAddress() string {
+	if os.Getenv("KONG_ADDRESS") != "" {
+		return os.Getenv("KONG_ADDRESS")
```

Comment:
Read once and return

addr := os.Getenv("foo")
if addr != "" {
  return addr
}